README.md

```diff
@@ -0,0 +1,18 @@
+# stw-submission-api
+
+The stw-submission-api repository is a ASP.NET Core Web API project using .NET 8.0 and controller based APIs that will be deployed on an Azure App Service to accept submissions from STW.
+
+## Running the API locally
+
+### Prerequisites
+If not already installed, you will need:
+
+- .NET 8 Runtime and SDK - Microsofts .NET 8 Downloads
+
+To run in locally in the stw-submission-api directory, execute this command in a terminal window:
+
+```bash
+dotnet run
+```
+
+Following the execution of this command your API should be running on port `5012`
```

Comment:
Is this port number correct? Looks like it should be 5244 from the launch settings